.github/workflows/e2e-matrix-builder.yml

```diff
@@ -29,12 +29,18 @@ jobs:
         shell: bash
         run: |
           echo "::set-output name=filenames::$(find e2e/test/scenarios -name '*.cy.spec.*' | tr '\n' ',')"
-      - run: yarn add underscore
       - uses: actions/github-script@v7
         id: matrix
         with:
           script: | # js
-            const _ = require('underscore');
+            const chunk = (arr, size) => {
+              const chunks = [];
+              const n = Math.ceil(arr.length / size);
+              for (let i = 0; i < n; i++) {
+                chunks.push(arr.slice(i * size, (i + 1) * size));
+              }
+              return chunks;
+            };
```

Comment:
without unit tests, I can't really parse if/how this will work for cases with unequal size chunks (e.g. if the last chunk would just be 1 item under the naive algorthm, do we make a few chunks 1 item smaller to balance them).  Maybe we could just use the raw [underscore code](https://underscorejs.org/docs/modules/chunk.html)?  

Also, today or tomorrow, I should be able to open a PR using cypress-split and remove this naive implementation and actually use spec timings.

Reply:
I actually took it from [here](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore?tab=readme-ov-file#_chunk) and tested to see if we get the exact same results. we currently parse 281 files. I can add/delete one to test if we handle even numbers as well.

Reply:
oops nevermind, I rewritten the code linked, but I still tested it and results are 1:1 with the old method